web/downloadService/get_public_filter_download_test.go

```diff
@@ -0,0 +1 @@
+package downloadService
```

Comment:
why a file with only one line?

Reply:
I committed these files by mistake - I have removed them from this branch 👍